content/en/agent/logs/proxy.md

```diff
@@ -15,8 +15,14 @@ further_reading:
 
 Log collection requires the Datadog Agent v6.0+. Older versions of the Agent do not include the `log collection` interface.
 
+## HTTPS log Forwarding
+
+**We recommend to [configure the Agent to send logs in HTTPS][2]** and to use the same [set of proxy settings][2] as other data types.
```

Comment:
The second link points to send logs over https rather than the proxy section (https://docs-staging.datadoghq.com/nils/log-proxy-instructions/agent/proxy)